content/pt-br/docs/concepts/policy/resource-quotas.md

```diff
@@ -0,0 +1,649 @@
+count/secrets                 1     4
+```
+
+## Capacidade e cota de Cluster
+
+`ResourceQuotas` são independentes da capacidade do cluster. Eles estão expresso em unidades absolutas. Portanto, se você adicionar nós ao cluster, isso *não*
+dá automaticamente a cada _namespace_ a capacidade de consumir mais recursos.
+
+Às vezes, políticas mais complexas podem ser necessárias, como:
+
+- Divida proporcionalmente os recursos totais do cluster entre várias equipes.
+- Permita que cada locatário aumente o uso de recursos conforme necessário, mas tenha um generoso limite para evitar o esgotamento acidental de recursos.
+- Detecte a demanda de um _namespace_, adicione nós e aumente a cota.
+
+Tais políticas podem ser implementadas usando `ResourceQuotas` como blocos de construção, por
+escrevendo um "controlador" que observa o uso da cota e ajusta os limites rígidos da cota de cada _namespace_ de acordo com outros sinais.
+
+Observe que a cota de recursos divide os recursos agregados do cluster, mas não cria restrições em torno dos nós: pods de vários _namespaces_ podem ser executados no mesmo nó.
+
+## Limite de consumo de classe de prioridade por padrão
+
+Pode ser desejado que os pods com uma prioridade particular, por exemplo. "cluster-services",
+deve ser permitido em um _namespace_, se, e somente se, existir um objeto de cota correspondente.
+
+Com este mecanismo, os operadores podem restringir o uso de certas classes de prioridade para um número limitado de _namespaces_ , e nem todos poderão consumir essas classes de prioridade por padrão.
+
+Para impor isso, a flag `kube-apiserver` `--admission-control-config-file` deve ser
+usada para passar o caminho para o seguinte arquivo de configuração:
+
+```yaml
+apiVersion: apiserver.config.k8s.io/v1
+kind: AdmissionConfiguration
+plugins:
+- name: "ResourceQuota"
+  configuration:
+    apiVersion: apiserver.config.k8s.io/v1
+    kind: ResourceQuotaConfiguration
+    limitedResources:
+    - resource: pods
+      matchScopes:
+      - scopeName: PriorityClass
+        operator: In
+        values: ["cluster-services"]
+```
+
+Em seguida, crie um objeto de cota de recurso no _namespace_ `kube-system`:
+
+{{< codenew file="policy/priority-class-resourcequota.yaml" >}}
+
+```shell
+kubectl apply -f https://k8s.io/examples/policy/priority-class-resourcequota.yaml -n kube-system
+```
+
+```none
+resourcequota/pods-cluster-services created
+```
+
+Nesse caso, a criação de um pod será permitida se:
+
+1. O `priorityClassName` do pod não foi especificado.
+1. O `priorityClassName` do pod é especificado com um valor diferente de `cluster-services`.
+1. O `priorityClassName` do pod está definido como `cluster-services`, ele deve ser criado no namespace `kube-system` e passou na verificação de cota de recursos.
+
+Uma solicitação de criação de pod é rejeitada se seu `priorityClassName` estiver definido como `cluster-services` e deve ser criado em um _namespace_ diferente de `kube-system`.
```

Comment:
Here, we can update the word "se" for "caso".
It should be: "é rejeitada caso seu"
Thanks

Reply:
Done! I will upload along with the other changes soon. Thanks for the suggestion!!